### PR TITLE
Adding to sniff branch

### PR DESCRIPTION
**Pull request by sniffowomo**
Adding thist test branch

### CODE REVIEW

**Comment by liqowomo** [comment]
Ok added the necessary details to this one

**Comment by sniffowomo** [comment]
Yes do that thang

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 <h1><code> Adding sniffowomo  </code></h1>
-<h2><i></i><h2>
+<h2><i> Another add</i><h2>
```
Comment (liqowomo) [inline README.md]: - This needs a major chnge

Comment (liqowomo) [reply README.md]: See if you can chane this

File: README.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
 <h1><code> Adding sniffowomo  </code></h1>
```
Comment (liqowomo) [inline README.md]: What about these heads , have a look at this

**Review by liqowomo — CHANGES_REQUESTED** [review]
Have a look at this pronto

File: README.md (hunk 1 above)
Comment (sniffowomo) [reply README.md]: Done niggers

File: README.md (hunk 2 above)
Comment (sniffowomo) [reply README.md]: my stink is strong

**Review by sniffowomo — COMMENTED** [review]
Danka

**Pull request merged**